app/controllers/prosecution_cases_controller.rb

```diff
@@ -5,14 +5,19 @@ class ProsecutionCasesController < ApplicationController
 
   add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
   add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
+  before_action :add_final_breadcrumb
```

Comment:
nitpick: I think this is not a special case, 
You could inline the breadcrumb here, or maybe put all the breadcrumbs inside the method
`before_action :add_breadcrumbs`